web_accounts_list.json

```diff
@@ -1748,6 +1759,17 @@
          "category" : "news",
          "valid" : true
       },
+      {
+         "name" : "Tik Tok",
+         "check_uri" : "https://www.tiktok.com/@{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : " Following",

```

Comment:
Can you find a better string for the account_existence_string? Something more direct to having a user profile.

Reply:
This is directly linked to having an account. And the json already uses the same method on instagram, periscope, delicious and smule ...

Reply:
And with regards to the other issue: Test it using Python instead of a browser. This should check out as proposed. It does in my end. Your browser must be translating the html entity. Remember a user agent - else you'll get json and not the html.